src/blog/2022/10/flowforge-1-released.md

```diff
@@ -0,0 +1,77 @@
+---
+title: FlowForge 1.0 released
+subtitle: FlowForge at 1.0, a huge milestone for us.
+description: FlowForge is now 1.0!
+date: 2022-10-27 18:00:00.0
+authors: ["rob-marcer"]
+video: XXXXXXXXXXXX
+---
+
+Auto assignment of unique IDs to Devices, define custom nodes to be pre-installed in your [Templates](https://flowforge.com/docs/user/concepts/#project-template) and import credentials files into your FlowForge [Projects](https://flowforge.com/docs/user/concepts/#project).
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 1.0 FlowForge is now available. Keep reading for a promotion code to get your first month free on FlowForge. Version 1.0 represents our vision of the base set of features needed for you to get great value from using FlowForge in a production environment. That's not to say we are done, we will continue to add features, improve our interfaces and fix bugs with the same enthusiasm as we've worked towards 1.0. We'd like to hear your feedback on what we will be including in [1.1 and beyond](https://github.com/orgs/flowforge/projects/5).
+
+## Features
+[Auto assignment of unique IDs to Devices](https://github.com/flowforge/flowforge/issues/841)
+We've updated FlowForge so that each running instance of Node-RED is assigned unique IDs. This can make handling of data returned from multiple [Devices](https://flowforge.com/docs/user/concepts/#device) far more robust.
+
+[Define custom Nodes to be pre-installed in your Templates](https://github.com/flowforge/flowforge/issues/405)
+This feature allows you to pre-define which custom nodes you want as part of your FlowForge [Templates](https://flowforge.com/docs/user/concepts/#project-template).
```

Comment:
```suggestion
[Add additional node modules to your projects](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define which custom nodes you want to be installed in your FlowForge project, making it easier to manage.
```

Reply:
```suggestion
[Define additional Nodes to be pre-installed](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define additional Node-RED nodes and node modules you may want as part of your FlowForge [Templates](https://flowforge.com/docs/user/concepts/#project-template).
```

Reply:
Given they are not part of the admin-created template, I'm wary of keeping saying template. I prefer my version.

Reply:
I made a booboo

My suggestion was supposed to remove the "templates" and "custom" nodes parts.  Custom suggests something other than an existing contrib or NPM module.

Reply:
```suggestion
[Add additional node modules to your projects](https://github.com/flowforge/flowforge/issues/405)
This feature allows you to pre-define additional Node-RED nodes and node modules you may want to be installed in your FlowForge project, making it easier to manage.
```

Reply:
that was what i meant to write DOH